### PR TITLE
Dotty 0.2.0-RC1 release announcement draft.

### PR DESCRIPTION
**Pull request by DarkDimius**

### CODE REVIEW

File: docs/blog/_posts/2017-07-11-second-dotty-milestone-release.md [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+     - [Fix conflict with  dragos-vscode-scala](https://github.com/lampepfl/dotty/pull/2777)
+     - [Fix ide crash on non-parsable file](https://github.com/lampepfl/dotty/pull/2752)
+     - [Fix hover functionality for enum classes](https://github.com/lampepfl/dotty/pull/2722)
+     - [Report errors on Dotty Language Server initialization](https://github.com/lampepfl/dotty/pull/2708)
+     - [Fixes to sbt setting up Dotty IDE](https://github.com/lampepfl/dotty/pull/2690)
+     - General stability improvements [2838](https://github.com/lampepfl/dotty/pull/2838), [2787](https://github.com/lampepfl/dotty/pull/2787), [2692](https://github.com/lampepfl/dotty/pull/2692)
+     
+  - Scalac compatibility improvements:
+  
+     - [Support Scala 2.12 traits](https://github.com/lampepfl/dotty/pull/2685)
```
Comment (smarter) [inline docs/blog/_posts/2017-07-11-second-dotty-milestone-release.md]: It's not actually working yet (https://github.com/lampepfl/dotty/pull/2827), so maybe not worth reporting yet.

File: docs/blog/_posts/2017-07-11-second-dotty-milestone-release.md [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+[Algebraic Data Types](http://dotty.epfl.ch/docs/reference/enums/adts.html),
+[By-Name Implicits](http://dotty.epfl.ch/docs/reference/implicit-by-name-parameters.html).
+
+This technology preview is geared towards improving stability and reliability. It includes:
+
+  - [Local optimizations upstreamed from Dotty Linker](https://github.com/lampepfl/dotty/pull/2513), [2647](https://github.com/lampepfl/dotty/pull/2647) by ([@OlivierBlanvillain](https://github.com/OlivierBlanvillain). See more details below.
+  - [Optimizing Pattern Matcher](https://github.com/lampepfl/dotty/pull/2829) by ([@odersky](https://github.com/odersky)
+  - [Idempotency checks](https://github.com/lampepfl/dotty/pull/2756) are the first step to reproducible builds
+  - [Faster Base class sets](https://github.com/lampepfl/dotty/pull/2676) by ([@odersky](https://github.com/odersky) and ([@darkdimius](https://twitter.com/darkdimius) 
+  - Huge number of fixes to IDE and Dotty Language Server covering:
```
Comment (smarter) [inline docs/blog/_posts/2017-07-11-second-dotty-milestone-release.md]: I think the most important IDE change that isn't mentioned in this list is that it now supports Windows: https://github.com/lampepfl/dotty/pull/2776

**Comment by liufengyun** [comment]
The CI for package works fine on Windows & Linux, Mac OS is queuing on Travis.  https://github.com/lampepfl/packtest/pull/5 .

**Comment by DarkDimius** [comment]
@liufengyun, thanks for the heads up!

**Comment by liufengyun** [comment]
Now the package test succeeds on all platforms https://github.com/lampepfl/packtest/pull/5.

The CI for brew is also green and I've merged: https://github.com/lampepfl/homebrew-brew/pull/4

New users can install the latest version with : `brew install lampepfl/brew/dotty`.

Existing users can use a shorter command: `brew upgrade dotty`.  If you try to run `brew install dotty` or `brew install lampepfl/brew/dotty`, Brew prints a friendly message saying `0.1.2-RC1` is already installed and `brew upgrade dotty` should be used.

File: docs/blog/_posts/2017-07-11-second-dotty-milestone-release.md (hunk 2 above)
Comment (DarkDimius) [inline docs/blog/_posts/2017-07-11-second-dotty-milestone-release.md]: Added.

File: docs/blog/_posts/2017-07-11-second-dotty-milestone-release.md (hunk 1 above)
Comment (DarkDimius) [inline docs/blog/_posts/2017-07-11-second-dotty-milestone-release.md]: Added to "next steps".

**Pull request merged**